doc/classes/NinePatchRect.xml

```diff
@@ -68,11 +68,11 @@
 		</constant>
 		<constant name="AXIS_STRETCH_MODE_TILE" value="1" enum="AxisStretchMode">
 			Repeats the center texture across the NinePatchRect. This won't cause any visible distortion. The texture must be seamless for this to work without displaying artifacts between edges.
-			[b]Note:[/b] Only supported when using the Vulkan renderer. When using the OpenGL renderer, this will behave like [constant AXIS_STRETCH_MODE_STRETCH].
+			[b]Note:[/b] Tiling is only supported when using the Forward+ or Mobile rendering methods. When using the Compatibility rendering method, this will behave like [constant AXIS_STRETCH_MODE_STRETCH].
```

Comment:
This is not intentional. The gl_compatibility renderer should behave the same as the others. Have you tested this?

Reply:
This documentation is likely a leftover from the GLES2 days. I've just tested this and can confirm it works as expected when using OpenGL.